apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.PostPolicy.cs

```diff
@@ -0,0 +1,315 @@
+            /// the <see cref="PostPolicyKnownElement.Key"/> element.
+            /// </summary>
+            BucketName = 1,
+
+            /// <summary>
+            /// The cache control you want to allow for the objects that will be uploaded.
+            /// </summary>
+            CacheControl = 2,
+
+            /// <summary>
+            /// The content disposition you want to allow for the objects that will be uploaded.
+            /// </summary>
+            ContentDisposition = 3,
+
+            /// <summary>
+            /// The content encoding you want to allow for the objects that will be uploaded.
+            /// </summary>
+            ContentEncoding = 4,
+
+            /// <summary>
+            /// The content length you want to allow for the objects that will be uploaded.
+            /// </summary>
+            ContentLength = 5,
+
+            /// <summary>
+            /// The content type you want to allow for the objects that will be uploaded.
+            /// </summary>
+            ContentType = 6,
+
+            /// <summary>
+            /// The date and time you want to allow before objects that will be uploaded
+            /// are considered stale by the browser.
+            /// The value for this element will be included in the <see cref="PostPolicy"/>
+            /// in ISO8601 basic format. When using this element for <see cref="PostPolicy.SetStartsWith(PostPolicyKnownElement, string)"/>
+            /// you should specify the part of the ISO8601 basic format representation that you want
+            /// values on upload to start with.
+            /// </summary>
+            ObjectExpiry = 7,
+
+            /// <summary>
+            /// The acceptable key name or a prefix for the uploaded objects.
+            /// This element should always be present in a post policy.
+            /// </summary>
+            Key = 8,
+
+            /// <summary>
+            /// The allowed URL to which the client is redirected upon successful upload.
+            /// </summary>
+            SuccessActionRedirect = 9,
+
+            /// <summary>
+            /// The status code returned to the client upon successful upload if 
+            /// <see cref="SuccessActionRedirect"/> is not specified.
+            /// Allowed values are <see cref="HttpStatusCode.OK"/>, <see cref="HttpStatusCode.Created"/>,
+            /// <see cref="HttpStatusCode.NoContent"/>.
+            /// </summary>
+            SuccessActionStatus = 10,
+        }
+
+        /// <summary>
+        /// A custom element that can be added to a <see cref="PostPolicy"/>.
+        /// </summary>
+        public enum PostPolicyCustomElement
+        {
+            /// <summary>
+            /// Custom element used for requiering extra Google metadata to be included
+            /// in the post request. If the element name you provide is not prefixed by
+            /// x-goog-meta-, it will be prefixed with x-goog-meta- before being added to
```

Comment:
`x-goog-meta-*` will be stored in Object resource custom metadata, for example `x-goog-meta-custom-1` will add a custom metadata key,value pair of `{"custom-1": "value"}`

Reply:
+1, specifically about the comment above not about the implementation.